doc/rados/operations/health-checks.rst

```diff
@@ -731,44 +744,42 @@ risk.  For example, if osd ``$N`` has the error:
 BLUESTORE_NO_COMPRESSION
 ________________________
 
-One or more OSDs is unable to load a BlueStore compression plugin.
-This can be caused by a broken installation, in which the ``ceph-osd``
-binary does not match the compression plugins, or a recent upgrade
-that did not include a restart of the ``ceph-osd`` daemon.
+One or more OSDs is unable to load a BlueStore compression plugin.  This issue
+might be caused by a broken installation, in which the ``ceph-osd`` binary does
+not match the compression plugins. Or it might be caused by a recent upgrade in
+which the ``ceph-osd`` daemon was not restarted.
 
-Verify that the package(s) on the host running the OSD(s) in question
-are correctly installed and that the OSD daemon(s) have been
-restarted.  If the problem persists, check the OSD log for any clues
-as to the source of the problem.
+To resolve this issue, verify that all of the packages on the host that is
+running the affected OSD(s) are correctly installed and that the OSD daemon(s)
+have been restarted. If the problem persists, check the OSD log for information
+about the source of the problem.
 
 BLUESTORE_SPURIOUS_READ_ERRORS
 ______________________________
 
-One or more OSDs using BlueStore detects spurious read errors at main device.
-BlueStore has recovered from these errors by retrying disk reads.
-Though this might show some issues with underlying hardware, I/O subsystem,
-etc.
-Which theoretically might cause permanent data corruption.
-Some observations on the root cause can be found at 
-https://tracker.ceph.com/issues/22464
+One or more BlueStore OSDs detect spurious read errors on the main device.
```

Comment:
Seems like an odd name, perhaps _transient_ would be better but it's baked in now.